lib/chef/resource/windows_task.rb

```diff
@@ -44,22 +44,22 @@ def initialize(name, run_context = nil)
       property :interactive_enabled, [TrueClass, FalseClass], default: false
       property :frequency_modifier, [Integer, String], default: 1
       property :frequency, equal_to: [:minute,
-                                       :hourly,
-                                       :daily,
-                                       :weekly,
-                                       :monthly,
-                                       :once,
-                                       :on_logon,
-                                       :onstart,
-                                       :on_idle,
-                                       :none], default: :hourly
+                                      :hourly,
+                                      :daily,
+                                      :weekly,
+                                      :monthly,
+                                      :once,
+                                      :on_logon,
+                                      :onstart,
```

Comment:
Any reason other than legacy that this symbol is inconsistent with the others? (i.e. ideally would be `:on_start`)

Reply:
Great question...